apps/examples/wakaama_client/wakaama_client.c

```diff
@@ -271,8 +193,7 @@ int wakaamaclient_main(int argc, char *argv[])
 				app_print_usage();
 				return 0;
 			}
-			if (1 != sscanf(argv[opt], "%d", \
-							&(dm_context.client_info.lifetime))) {
+			if (1 != sscanf(argv[opt], "%d", &(dm_context.client_info.lifetime))) {
```

Comment:
Please modify to if (sscanf() != 1)
From now, please apply this style when you implement something